src/vsr.zig

```diff
@@ -491,7 +526,7 @@ pub const Header = extern struct {
         if (self.client != 0) return "client != 0";
         if (self.request != 0) return "request != 0";
         if (self.op != 0) return "op != 0";
-        if (self.timestamp != 0) return "timestamp != 0";
+        if (self.timestamp == 0) return "timestamp == 0";
```

Comment:
Since `commit` messages indicate to backups that the primary is alive, they use `timestamp` to discard old (or repeated) heartbeats.